man/systemd.network.xml

```diff
@@ -1050,7 +1050,7 @@
           <listitem>
             <para>When true, the interface maximum transmission unit
             from the DHCP server will be used on the current link.
-            Defaults to false.</para>
+            Prior to v234 this setting defaulted to false. Defaults to true.</para>
```

Comment:
we don't document legacy behaviour in our docs, except for NEWS. If people are looking for that they should check the docs of such old versions. We really don#t want to carry around all that old cruft forever... please remove this and simply document the new behaviour in the man pages.

And please add a NEWS entry about this.

Reply:
OK